mautic/mautic
# DPMMA-2697 Tag usages

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🟢 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description
This PR adds information about Tag usage in segments, campaigns, forms, points and reports. 

Similar functionality already works for Segments and Emails.

![image](https://github.com/mautic/mautic/assets/8580942/2d9999d2-a3e3-481b-a5b3-3c1679a3e5ee)


---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Add a new Tag
3. Use the Tag in segment filter / campaign action / point trigger / report
4. Check that the Tag usage is visible in the Tag view
